pkg/apis/common/v1/association.go

```diff
@@ -92,6 +92,10 @@ func (asm AssociationStatusMap) AllEstablished() bool {
 const (
 	ElasticsearchConfigAnnotationNameBase = "association.k8s.elastic.co/es-conf"
 	ElasticsearchAssociationType          = "elasticsearch"
+	MetricbeatConfigAnnotationNameBase    = "association.k8s.elastic.co/mb-conf"
+	MetricbeatAssociationType             = "metricbeat"
+	FilebeatConfigAnnotationNameBase      = "association.k8s.elastic.co/fb-conf"
+	FilebeatAssociationType               = "filebeat"
```

Comment:
Do we need two separate controllers? I would imagine that the Beats users should be compatible for both?

Reply:
Yes, only one is enough.